### PR TITLE
Remove {{/* */}} comments from error constants in cse_helpers.sh

### PR DESCRIPTION
**Pull request by anujmaheshwari1**
The {{/* */}} comments following the timeout error constants in cse_helpers.sh cause the constants to not be properly set in the bash environment. If we ever hit one of these constants in the VHD Build, they will throw out an unbound variable error. 

Fixed this using sed to remove the comments before sourcing the files. 

Also moved chrony/ntp/timesyncd related constants to tool_installs_ubuntu.sh

### CODE REVIEW

**Review by eriksywu — APPROVED** [review]
Thanks Arun!

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -4,6 +4,9 @@ OS=$(sort -r /etc/*-release | gawk 'match($0, /^(ID_LIKE=(coreos)|ID=(.*))$/, a)
 UBUNTU_OS_NAME="UBUNTU"
 MARINER_OS_NAME="MARINER"
 
+sed -i 's/{{\/\*[^*]*\*\/}}//g' /home/packer/provision_source.sh
```
Comment (eriksywu) [inline vhdbuilder/packer/install-dependencies.sh:7]: small nit: add comment to explain what the sed is doing

File: vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh [hunk 2]
```diff
@@ -9,6 +9,13 @@ ERR_UA_DETACH=185 {{/* Error to detach UA */}}
 ERR_LINUX_HEADER_INSTALL_TIMEOUT=186 {{/* Timeout to install linux header */}}
 ERR_STRONGSWAN_INSTALL_TIMEOUT=187 {{/* Timeout to install strongswan */}}
 
+ERR_NTP_INSTALL_TIMEOUT=10 {{/*Unable to install NTP */}}
```
Comment (xuto2) [inline vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh:12]: tool_installs_ubuntu.sh is not part of cse, so no need to use the {{ }} template syntax. We can just add # before the comment

**Review by xuto2 — APPROVED** [review]

File: vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh (hunk 2 above)
Comment (anujmaheshwari1) [reply vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh:12]: should I update the other constants(FIPS related error codes) in tool_installs_ubuntu.sh declared above the ntp/timesyncd/chrony similarly or leave them as it is?

**Pull request merged**